plugin/audit_log/audit_log.c

```diff
@@ -1746,7 +1748,9 @@ static struct st_mysql_audit audit_log_descriptor=
 
 static struct st_mysql_show_var audit_log_status_variables[]=
 {
-  { 0, 0, 0}
+  {"Audit_buffer_size_too_small",
```

Comment:
Should start with plugin name prefix (which is "audit_log")
"too_small" does not make it obvious that this is a counter, please consider other names